### ISSUE TITLE
can't add metadata

### ISSUE THREAD

**Issue by charlesthomas** [caarlos0/svu#88]
there are flags to strip metadata, but there doesn't appear to be a way to add metadata

`--suffix` flag will always use a `-` separator

**Comment by mamachanko** [comment]
Agreed!

I would want, say `v0.0.0+nightly`, but I get:

```shell
❯ svu --suffix +nightly
v0.0.0-+nightly

❯ svu --suffix nightly
v0.0.0-nightly
```

**Comment by caarlos0** [comment]
we should probably deprecate `--suffix` and be more specific, there are pre-release fields AND build fields on semver:

https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

**Comment by caarlos0** [comment]
will be fixed by #101
